CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ include(PeleCUtils)
 
 #General options for all executables in the project
 set(PELEC_DIM "3" CACHE STRING "Number of physical dimensions")
+option(PELEC_ENABLE_AMREX_PARTICLES "Enable particles in AMReX library" OFF)
```

Comment:
Can we use `PELEC_ENABLE_SPRAY` here? I think it's up to you whether or not to consider the work as sprays or particles in general, but I would just want to be consistent.

Reply:
Yeah. As of right now, PeleC only has spray particles in my specific branch. But PeleLM has separate particle functionality that I tried to preserve from things in IAMR. In general, I want to have a separate flag for sprays in case we want other particles. So I will change that flag to reflect that.

Reply:
Ok, how about we just use `PELEC_ENABLE_PARTICLES` for this and use that to turn on the AMReX particles?

Reply:
Actually it's fine as it is. I think we will need to treat it like we did with the EB.

Reply:
Ok. I think I got it. I will do another commit soon. Thanks!

Reply:
I think we will want to build AMReX always with particles enabled like we did with EB, so CMake only needs to build AMReX once. Then we can turn particles on or off in PeleC executables if that makes sense. This is only a CMake issue though.